src/ralph/admin/templates/admin/login.html

```diff
@@ -2,8 +2,8 @@
 {% load i18n admin_static %}
 
 {% block extrastyle %}
-  {{ block.super }}
-  {{ form.media }}
+{{ block.super }}
```

Comment:
The way django renders template make output html messy - adding innecessary spaces. 
Do we really want to make this shit happens?